test/helpers.ts

```diff
@@ -0,0 +1,15 @@
+import { Contract } from 'ethers';
```

Comment:
What is this helper used for?

Reply:
it's used in the test to initialize staticatoken's proxy, see [here](https://github.com/aave-starknet-project/aave-starknet-bridge/pull/13/files#diff-c52ab99d41f7be91138f669dc6d12d85892a70aeb50f5ed4229e594c4f20b2ffR185) for instance